src/scramBag.js

```diff
@@ -0,0 +1,13 @@
+const bagOfTiles = { 'A':9, 'B':2, 'C':2, 'D':4, 'E':12, 'F':2, 'G':3, 'H':2, 'I':9, 'J':1, 'K':1, 'L':4, 'M':2, 'N':6, 'O':8, 'P':2, 'Q':1, 'R':6, 'S':4, 'T':6, 'U':4, 'V':2, 'W':2, 'X':1, 'Y':2, 'Z':1, '_':2 }
+
+export const scramBag = (input) => {
+  const stringArray = input.split('').join(',')
+  for(let i in bagOfTiles) {
+    for(let j in stringArray) {
```

Comment:
this for look shouldn't work. `stringArray` is not an array. `.join(',')` returns a string.